go.mod

```diff
@@ -28,10 +28,17 @@ require (
 	github.com/Azure/go-autorest/tracing v0.1.0 // indirect
 	github.com/GoogleCloudPlatform/cloudsql-proxy v0.0.0-20190418212003-6ac0b49e7197
 	github.com/Shopify/sarama v1.19.0
+	github.com/Shopify/toxiproxy v2.1.4+incompatible // indirect
 	github.com/aws/aws-sdk-go v1.19.16
+	github.com/coreos/bbolt v1.3.2 // indirect
```

Comment:
Any idea where this is coming from now? What does go mod why -m say?

Reply:
```
gocloud.dev/runtimevar/etcdvar
go.etcd.io/etcd/clientv3
go.etcd.io/etcd/clientv3.test
github.com/coreos/etcd/auth
github.com/coreos/etcd/mvcc/backend
github.com/coreos/bbolt
```

Reply:
Ugh, but your PR doesn't even update the etcd version. What a mess :/